mne/channels/montage.py

```diff
@@ -23,17 +23,18 @@
 from ..io.pick import pick_types
 from ..io.open import fiff_open
 from ..io.constants import FIFF
-from ..utils import _check_fname, warn
+from ..utils import _check_fname, warn, copy_function_doc_to_method_doc
```

Comment:
omg, this function idea is amazing! and has been needed! wait, is this used for objects to read methods as well?